fresque/__init__.py

```diff
@@ -10,15 +10,17 @@
 import logging.handlers
 import os
 import sys
-
 import flask
 import flask_fas_openid
-
+from inspect import getmembers, isfunction
 
 APP = flask.Flask(__name__)
+APP.jinja_env.trim_blocks = True
+APP.jinja_env.lstrip_blocks = True
```

Comment:
You may want to consider if you want this or not (project wide, you can also do it locally for loop or if) as it may make the html produced quite un-readable